libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Scopes/MemoryScope.cs

```diff
@@ -59,7 +59,7 @@ public MemoryScope(string name, bool includeInSnapshot = true)
         /// <param name="cancellationToken">A cancellation token that can be used by other objects
         /// or threads to receive notice of cancellation.</param>
         /// <returns>A task that represents the work queued to execute.</returns>
-        /// <exception cref="ArgumentNullException"><paramref name="turnContext"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentNullException">.</exception>
```

Comment:
does it make sense to leave the exception tag? will this ever now throw an exception? Seems like the ArgumentNullException referred to a turnContext that no longer exists

Reply:
all the exceptions tags has been removed